screens/signup/index.js

```diff
@@ -0,0 +1,207 @@
+  render() {
+    return (
+      <View style={styles.container}>
+        <Image source={background} style={styles.bg} resizeMode="cover" />
+        <View style={styles.headerIconView}>
+          <TouchableOpacity style={styles.headerBackButtonView}>
+            <Image source={backIcon} style={styles.backButtonIcon} />
+          </TouchableOpacity>
+        </View>
+        <View style={styles.headerTitleView}>
+          <Text style={styles.titleViewText}>Sign Up</Text>
+        </View>
+        <View style={styles.inputs}>
+          <View style={styles.inputContainer}>
+            <View style={styles.imageContainer}>
+              <Image source={personIcon} style={styles.inputIcon} />
+            </View>
+            <TextInput
+              style={[styles.input, styles.whiteFont]}
+              placeholder="Name"
+              placeholderTextColor="#FFF"
+              underlineColorAndroid='transparent' />
+          </View>
+          <View style={styles.inputContainer}>
+            <View style={styles.imageContainer}>
+              <Image source={emailIcon} style={styles.inputIcon} />
+            </View>
+            <TextInput
+              style={[styles.input, styles.whiteFont]}
+              placeholder="Email"
+              placeholderTextColor="#FFF" />
+          </View>
+          <View style={styles.inputContainer}>
+            <View style={styles.imageContainer}>
+              <Image source={lockIcon} style={styles.inputIcon} />
+            </View>
+            <TextInput
+              secureTextEntry={true}
+              style={[styles.input, styles.whiteFont]}
+              placeholder="Password"
+              placeholderTextColor="#FFF" />
+          </View>
+          <View style={styles.inputContainer}>
+            <View style={styles.imageContainer}>
+              <Image source={birthdayIcon} style={styles.inputIcon} />
+            </View>
+            <TextInput
+              style={[styles.input, styles.whiteFont]}
+              placeholder="Birthday"
+              placeholderTextColor="#FFF"
+              underlineColorAndroid='transparent' />
+          </View>
+        </View>
+        <View style={styles.footerContainer}>
+          <View style={styles.signupView}>
+            <TouchableOpacity
+              style={styles.signup}>
+              <Text style={styles.signupText}>Join</Text>
+            </TouchableOpacity>
+          </View>
+          <TouchableOpacity
+            style={styles.signin}>
+            <Text style={styles.greyFont}>Already have an account?<Text style={styles.whiteFont}> Sign In</Text></Text>
+          </TouchableOpacity>
+        </View>
+      </View>
+    );
+  }
+}
+
+let styles = StyleSheet.create({
+  container: {
+    flexDirection: 'column',
+    flex: 1,
+    backgroundColor: 'transparent'
+  },
+  bg: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    width: Dimensions.get('window').width,
+    height: Dimensions.get('window').height
```

Comment:
This is part has been revamped in latest code, Checkout the new code structure of the login page